### PR TITLE
[WIP] Bump ex_unit assertion time to 200ms

### PR DESCRIPTION
**Pull request by yurikoval**

### CODE REVIEW

File: config/test.exs [hunk 1]
```diff
@@ -188,3 +188,5 @@ config :exvcr,
 config :echo_boy, port: 4100
 
 config :ex_bitmex, domain: "testnet.bitmex.com"
+
+config :ex_unit, assert_receive_timeout: 200
```
Comment (rupurt) [inline config/test.exs]: 😎 if this is the problem. Thanks for looking into it.

How do you feel about making this configurable as an env var with a default?

Comment (yurikoval) [reply config/test.exs]: This test is failing on CI only. I can extract it once I can confirm this the cause.

**Comment by rupurt** [comment]
I think we're going to need to configure storage for artifacts to debug this. There must be some useful info the logs.

https://circleci.com/docs/2.0/artifacts/

**Comment by yurikoval** [comment]
CI failed with 10s `assert_receive_timeout`, so I think we can rule that out.

**Pull request closed without merging**